go/oasis-node/cmd/debug/txsource/workload/governance.go

```diff
@@ -54,6 +54,7 @@ type governanceWorkload struct {
 	ctx context.Context
 	rng *rand.Rand
 
+	beacon       beacon.Backend
```

Comment:
Could get this from `g.Consensus().Beacon()` now. (Also elsewhere.)

Reply:
`./governance.go:336:29: g.BaseWorkload.Consensus().Beacon undefined (type "github.com/oasisprotocol/oasis-core/go/consensus/api".ClientBackend has no field or method Beacon)`